src/app/(pages)/wdi/page.tsx

```diff
@@ -59,77 +85,16 @@ export default function Page() {
 			<div className="mb-8">
 				<h2 className="mb-4 text-2xl font-semibold">Event Details</h2>
 				<ul className="space-y-3">
-					<li className="flex items-start">
-						<span className="bg-blue-100 mr-3 inline-flex items-center justify-center rounded-full p-1">
-							<svg
-								xmlns="http://www.w3.org/2000/svg"
-								className="text-blue-600 h-5 w-5"
-								fill="none"
-								viewBox="0 0 24 24"
-								stroke="currentColor"
-							>
-								<path
-									strokeLinecap="round"
-									strokeLinejoin="round"
-									strokeWidth={2}
-									d="M8 7V3m8 4V3m-9 8h10M5 21h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v12a2 2 0 002 2z"
-								/>
-							</svg>
-						</span>
-						<span>
-							<strong>Dates:</strong> April 4th, 2025 (online) and April 5th,
-							2025 (in-person)
-						</span>
-					</li>
-					<li className="flex items-start">
-						<span className="bg-blue-100 mr-3 inline-flex items-center justify-center rounded-full p-1">
-							<svg
-								xmlns="http://www.w3.org/2000/svg"
-								className="text-blue-600 h-5 w-5"
-								fill="none"
-								viewBox="0 0 24 24"
-								stroke="currentColor"
-							>
-								<path
-									strokeLinecap="round"
-									strokeLinejoin="round"
-									strokeWidth={2}
-									d="M17.657 16.657L13.414 20.9a1.998 1.998 0 01-2.827 0l-4.244-4.243a8 8 0 1111.314 0z"
-								/>
-								<path
-									strokeLinecap="round"
-									strokeLinejoin="round"
-									strokeWidth={2}
-									d="M15 11a3 3 0 11-6 0 3 3 0 016 0z"
-								/>
-							</svg>
-						</span>
-						<span>
-							<strong>Location:</strong> Faculty of Mathematics and Information
-							Science, Warsaw University of Technology
-						</span>
-					</li>
-					<li className="flex items-start">
-						<span className="bg-blue-100 mr-3 inline-flex items-center justify-center rounded-full p-1">
-							<svg
-								xmlns="http://www.w3.org/2000/svg"
-								className="text-blue-600 h-5 w-5"
-								fill="none"
-								viewBox="0 0 24 24"
-								stroke="currentColor"
-							>
-								<path
-									strokeLinecap="round"
-									strokeLinejoin="round"
-									strokeWidth={2}
-									d="M19 11H5m14 0a2 2 0 012 2v6a2 2 0 01-2 2H5a2 2 0 01-2-2v-6a2 2 0 012-2m14 0V9a2 2 0 00-2-2M5 11V9a2 2 0 012-2m0 0V5a2 2 0 012-2h6a2 2 0 012 2v2M7 7h10"
-								/>
-							</svg>
-						</span>
-						<span>
-							<strong>Tracks:</strong> 25 tracks with 300 talks
-						</span>
-					</li>
+					{details.map(({ name, icon: Icon, description }) => (
+						<li key={name} className="flex items-start">
```

Comment:
we need to change tab to 2 spaces :P this code is unreadable in code review

Reply:
Yup, I've noticed that on GH as well 😅